packages/yoga/src/Dialog/web/Dialog.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useRef, useCallback } from 'react';
+import { createPortal } from 'react-dom';
+import styled from 'styled-components';
+import { func, bool, node } from 'prop-types';
+
+import { usePortal } from '../../hooks';
+import { Button, Card, Box } from '../..';
+import { Close } from '../../../../icons/src';
+
+const StyledDialog = styled(Card)`
+  ${({
+    onClose,
+    theme: {
+      yoga: {
+        components: { dialog },
+      },
+    },
+  }) => `
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+
+  padding: ${onClose ? dialog.padding.withIconClose : dialog.padding.top}px 
+  ${dialog.padding.default}px 
+  ${dialog.padding.default}px;
+
+  width: ${dialog.width.default}px;
+  min-height: ${dialog.height.min}px;
+  border-radius: ${dialog.border.radius}px;
+
+  backdrop-filter: blur(1px);
```

Comment:
This isn't well supported in [some browsers](https://caniuse.com/css-backdrop-filter)